ui/raidboss/data/timelines/ridorana_lighthouse.txt

```diff
@@ -34,10 +34,10 @@ hideall "--sync--"
 176 "Dark Cannonade" sync /:Famfrit, the Darkening Cloud:2C42:/
 200 "Water IV" sync /:Famfrit, the Darkening Cloud:2C3D:/
 208 "Tsunami" sync /:Famfrit, the Darkening Cloud:2C50:/
-214 "Tsunami" duration 3
-228 "Tsunami" duraiton 3
```

Comment:
I've been thinking about this a lot, and I really need to get some bots to verify timelines and trigger syntax on pull requests (and master).  I need to automate this for sure with so many contributions from you and other folks.